app/src/main/java/com/example/habithelper/fragments/HomeFragment.java

```diff
@@ -19,20 +17,16 @@
 import android.widget.Button;
 import android.widget.ProgressBar;
 import android.widget.TextView;
-
 import com.codepath.asynchttpclient.AsyncHttpClient;
 import com.codepath.asynchttpclient.callback.JsonHttpResponseHandler;
 import com.example.habithelper.activities.LoginActivity;
 import com.example.habithelper.utilities.LinearRegressionCalculator;
-import com.example.habithelper.utilities.OLSMultipleLinearRegression;
 import com.example.habithelper.R;
 import com.example.habithelper.models.TrackDay;
 import com.parse.FindCallback;
 import com.parse.ParseException;
 import com.parse.ParseQuery;
 import com.parse.ParseUser;
-
```

Comment:
Not specific for this line, but from the previous PR it left pending the sanity check of null and the reason to call the ParseUser.getCurrentUser(); after logout. 
Let's file an issue to track those. Not blocking the PR but seems it would be pending actions for quality purpose